test/test_util_scopes.js

```diff
@@ -11,8 +11,8 @@ suite('test lib/util/scopes', function() {
     ['starScope',   'pfx:', { vol1: '/vol1', vol2: '/vol2' }, ['pfx:*'], true],
     ['noResources', 'pfx:', {}, [], true],
   ].forEach((t)=> {
```

Comment:
@walac, this should probably be a `map` and wrapped in a `await Promise.all(...`
And then `suite(...., async function() {` would probably be a good idea...

Reply:
It's just calling test(..) with a function argument -- it's up to the test runner to wait for the promises.

Reply:
ah, I'm not even sure it works on the suite level...